### PR TITLE
coredump: raise limits

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

**Review by yuwata — APPROVED** [review]
LGTM.

File: src/coredump/coredump.c [hunk 1]
```diff
@@ -48,8 +48,14 @@
 #include "uid-alloc-range.h"
```
Comment (dnicolodi) [inline src/coredump/coredump.c:48]: I think in the commit message "4 orders of magnitude" should be "16 times" as orders of magnitude usually are intended in base 10 😄

Comment (poettering) [reply src/coredump/coredump.c:48]: Are you sure that's the case?

I mean https://en.wikipedia.org/wiki/Order_of_magnitude says "usually 10", but then in the introductory paragraphs already points to the fact that one might use "2" as base in computers...

**Comment by kgibm** [comment]
Thank you!

**Pull request merged**